packages/terra-list/src/ListItem.jsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import classNames from 'classnames';
+import classNames from 'classnames/bind';
 import 'terra-base/lib/baseStyles';
 import Arrange from 'terra-arrange';
 import ChevronRight from 'terra-icon/lib/icon/IconChevronRight';
-import './ListItem.scss';
```

Comment:
There is hard dependency between previous List.scss and ListItem.scss file. As the divided border style is assigned to `list.divided > .list-item` by using `list-item` class, and list item will add border color to `.list-item` if it's inside the `divided list`. So only keep one list.scss file to provide one scope for two react components.